runtime/lua/vim/ui.lua

```diff
@@ -117,6 +120,8 @@ end
 --- -- Asynchronous.
 --- vim.ui.open("https://neovim.io/")
 --- vim.ui.open("~/path/to/file")
+--- -- Command 'osurl' knows what to do with this reference
+--- vim.ui.open("gh#neovim/neovim!29490", "osurl")
```

Comment:
is `osurl` something we should add as a fallback to the builtin impl?

Reply:
No, it is just [my script](https://git.sr.ht/~mcepl/vim-suse-changes/tree/master/item/osurl) which handles [SUSE abbreviated URLs](https://en.opensuse.org/openSUSE:Packaging_Patches_guidelines#Current_set_of_abbreviations). It is meant just as an example of a random script.